### DISCUSSION TITLE
How to work with issues that should touch api and cli?

### DISCUSSION THREAD

**Discussion by StefMa** [ioki-mobility/go-outline#28] [General]
The repository contains an api client as well as an cli client.

I ask myself how we should organize issues for those "two different projects".

Example:
If we want to add a new api, we probably also want to support this in the cli.
Should we, in that case, create a single issue that is there to handle the cli as well as the api? Or should we create two (similar) issues while one is there to adds the support for the api and the other one for the cli?!

In my opinion we should create two tickets. Because even if they are close to each other, both clients are not the same. We may have to discuss different things how stuff look like.
Additionally, if we have two separated tickets, the barrier to create a single PR that adds support for both clients, is less (which I think is a good thing).

What do you think?

Tip:
To distinguish issues that may have the same title, we could create labels like "in:cli" and "in: api" which we add to those issues respectively.

**Comment by rsjethani** [comment]
The api and cli clients are  related yet different products. However they should not be tied together and should grow independently at their own pace. The client can have lot more features than the cli and it is okay. 

Hence an issue for adding new feature into client should have nothing to do with the cli. Only cases where both products are touched would be say a bug fix or api change which affects both products.

Yes we can label the issues as @StefMa suggests.
